projects/aca-content/src/lib/components/files/files.component.spec.ts

```diff
@@ -156,6 +156,14 @@ describe('FilesComponent', () => {
 
       expect(router.navigate['calls'].argsFor(0)[0]).toEqual(['/personal-files', 'parent-id']);
     });
+
+    it('should show the message "Your filter returned 0 results" when filter(s) are selected and there are no records', () => {
+      Object.defineProperty(route, 'queryParamMap', { value: of({ params: { $thumbnail: 'TYPE:"cm:folder"' } }) });
+
+      fixture.detectChanges();
+
+      expect(component.isFilterHeaderActive).toBeTrue();
```

Comment:
is this a joke? how the test is performing what it claims to do in the test name?

Reply:
The flag 'isFilterHeaderActive', which is being checked to be true, is used to decide weather the message, mentioned in the test name, should be visible or not. The filters are being fetched from queryParamMap. Still, I have updated the name of the unit test, further.